LibTrixi.jl/src/LibTrixi.jl

```diff
@@ -24,9 +25,12 @@ export trixi_step,
 export trixi_ndims,
        trixi_ndims_cfptr,
        trixi_ndims_jl
-export trixi_nelements,
-       trixi_nelements_cfptr,
-       trixi_nelements_jl
+export trixi_nelements_local,
+       trixi_nelements_local_cfptr,
+       trixi_nelements_local_jl
```

Comment:
Without going into more details here, I'd recommend to not call it `nelements_local` but to stick to `nelements` and `nelements_global`. Most codes (that I know) are written such that they assume that all values are local unless noted otherwise, since otherwise you'd have to add the `_local` suffix in too many places.